CHANGELOG/CHANGELOG-3.0.md

```diff
@@ -0,0 +1,386 @@
+# Release notes for v3.0.0
```

Comment:
It looks like there's a number of PRs that should have release notes.

These didn't specify a release note but I think is important to include:
https://github.com/kubernetes-csi/external-provisioner/pull/646

These do have a release note but were ot picked up by the tool for some reason?
https://github.com/kubernetes-csi/external-provisioner/pull/636
https://github.com/kubernetes-csi/external-provisioner/pull/635
https://github.com/kubernetes-csi/external-provisioner/pull/620
https://github.com/kubernetes-csi/external-provisioner/pull/617

Reply:
@Jiawei0227 can you help word a release note for #646? This looks related to migration

Reply:
I think this was a byproduct of me trying to invoke `release-notes` with specific SHAs vs. using `--discover=mergebase-to-latest`, I was having issues with it last time I invoked it. Will try again.

Reply:
for #646 

```
Newly provisioned CSI Migration enabled PV will have "provisioned-by" annotation set to in-tree provisioner name instead of the CSI provisioner
```